controllers/vrg_volrep.go

```diff
@@ -647,6 +683,21 @@ func (v *VRGInstance) reconcileVRForDeletion(pvc *corev1.PersistentVolumeClaim,
 		}
 	}
 
+	return v.undoPVCFinalizersAndPVRetention(pvc, log)
+}
+
+func (v *VRGInstance) undoPVCFinalizersAndPVRetention(pvc *corev1.PersistentVolumeClaim, log logr.Logger) bool {
+	const requeue = true
+
+	pvcNamespacedName := types.NamespacedName{Name: pvc.Name, Namespace: pvc.Namespace}
+
+	if pvc.GetAnnotations() != nil && pvc.GetAnnotations()[RestoreAnnotation] == RestoredByRamen {
+		// We created the PVC, delete it
```

Comment:
When disabling an app for DR, we would be deleting the VRG as Primary on one end, at which point we should not delete the PVC and neither wait for its deletion.

When deleting an app and hence deleting the VRG, deleting the PVCs is acceptable, but at present we cannot distinguish between the 2 in VRG

Reply:
Discussed with @BenamarMk we will add some layer of defense for this potentially as follows,
- Initially do not support disable, hence the current workflow is clean
- Subsequently support disable workflow, at which point the hub, based on Placement deletion and Placement ownership (by DRPC or otherwise), would instruct when to garbage collect the PVCs

The above would be done subsequently, and need not block the current merge.

Reply:
Issue created here: https://github.com/RamenDR/ramen/issues/833